app/models/post.rb

```diff
@@ -3,7 +3,11 @@ class Post < ApplicationRecord
   has_many :comments, dependent: :destroy
   has_many :likes, dependent: :destroy
 
-  after_save :update_post_counter
+  validates :author, presence: true
+  validates :title, presence: true, length: { maximum: 250 }
+  validates :comments_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+  validates :likes_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+  validtaes :posts_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
```

Comment:
- [ ] Good job for adding the validations for user model 👏 However, the previous reviewer didn't requested to add the validations for `author` and `posts_counter` in `post.rb` . Also, according to the project requirements these validations are not required here, it is also affecting our tests 🐛 Currently, I'm getting the following error⚠ Since the `author` and `posts_counter` validations in `post` model were not present during the last review and are not requested either, therefore kindly rectify this change.

![image](https://github.com/Nessrine88/Blog-App/assets/106119105/925c09c0-73db-44d7-a017-0f76dea83dbe)

_Project requirements_
![image](https://github.com/Nessrine88/Blog-App/assets/106119105/e2d73ad8-7242-4d77-b374-475892254798)